pkg/cli/start.go

```diff
@@ -744,8 +744,10 @@ func getDefaultGoMemLimit(ctx context.Context) int64 {
 		log.Ops.Shoutf(
 			ctx, severity.WARNING, "recommended default value of "+
 				"--max-go-memory (%s) was truncated to %s, consider reducing "+
-				"--max-sql-memory and / or --cache",
+				"--max-sql-memory (%s) and / or --cache (%s); total system/cgroup memory: %s.",
```

Comment:
The args are swapped - you want the memory pool size first (since that's the sql mem according to line line 739 above), then `serverCfg.CacheSize` since that's the `--cache`, then `sysMem`.

Reply:
Good catch, thanks!